bin/test

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+set -euf -o pipefail
+
+# shellcheck source=functions
+source "${BASH_SOURCE%/*}/../sanctuary-scripts/functions"
+
+if [[ "${RECURSING-false}" == false ]] && [[ -x scripts/test ]] ; then
+  RECURSING=true scripts/test
+  exit 0
+fi
+
+branches="$(get min-branch-coverage)"
+
+node_modules/.bin/istanbul cover node_modules/.bin/_mocha
+node_modules/.bin/istanbul check-coverage --branches "$branches"
```

Comment:
Did you consider updating to [`nyc`](https://github.com/istanbuljs/nyc)?

Reply:
No. Do you think we should do so? What are the advantages?

Reply:
Let's wait for a tool that uses V8 native code coverage reporting.

Reply:
It just spawned into existence https://github.com/bcoe/c8